src/Native/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -334,7 +334,7 @@ int UTF8ToWideChar(const char* bytes, int len, wchar_t* buffer, int bufLen)
     size_t inbufbytesleft = len;
     size_t outbufbytesleft = bufLen;
 
-    int rc = iconv(cd, &inbuf, &inbufbytesleft, &outbuf, &outbufbytesleft);
+    int rc = iconv(cd, const_cast<const char **>(&inbuf), &inbufbytesleft, &outbuf, &outbufbytesleft);
```

Comment:
I remember that the signature of iconv is not the same on all platforms. Some declaring `const char **', some just `char **'. Linux is `char **'.